config/config.go

```diff
@@ -1,11 +1,19 @@
 package config
 
 import (
+	"github.com/ONSdigital/dp-feedback-api/sdk"
 	"time"
 
 	"github.com/kelseyhightower/envconfig"
 )
 
+// Config for feedback sdk
+type FeedbackConfig struct {
```

Comment:
if you want to have a separate config structure it's good (actually I think it makes a lot of sense), 
but then you need to add a field to the Config struct (comment below)